Sample-01/server.js

```diff
@@ -8,7 +8,13 @@ const app = express();
 const port = process.env.SERVER_PORT || 3000;
 
 app.use(morgan("dev"));
-app.use(helmet());
+
+app.use(
+  helmet({
+    contentSecurityPolicy: false,
+  })
+);
```

Comment:
This PR upgrades `helmet` from 3 to 4. In 3, `contentSecurityPolicy` was disabled by default, but _enabled_ by default in 4. I've disabled it here to get the same behaviour as it's probably overkill for a sample anyway. Either way, it breaks our integration tests as the site is just plain broken when executed using `node server.js`, as it would be when running inside Docker or outside of `react-scripts`.